From a0ed85e0b268dba76d9c820571e1b9217b51e985 Mon Sep 17 00:00:00 2001
From: "Hongzhi.Song" <hongzhi.song@windriver.com>
Date: Tue, 12 Mar 2019 20:03:21 -0700
Subject: [PATCH] glusterd: change port range

glusterfsd chooses tcp ports within the dynamic
range [49152-65535] to provide its service.  When
libvirt performs migration it indicates qemu
to use tcp port within dynamic range [49152-49216]
as well.

When both glusterfsd and qemu/libvirt use the same
port (e.g. 49152) then libvirtd crashes and
causes migration failed.

Modify the base-port for glusterfsd starting from
59152 to avoid port conflict with qemu/libvirt

Upstream-Status: Pending

Signed-off-by: Vu Tran <vu.tran@windriver.com>
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
[Yi: minor adjust to apply the patch on 3.11.1]
Signed-off-by: Yi Zhao <yi.zhao@windriver.com>
[Hogzhi: minor adjust to apply the patch on v5.4]
Signed-off-by: Hongzhi.Song <hongzhi.song@windriver.com>
---
 extras/glusterd.vol | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/extras/glusterd.vol b/extras/glusterd.vol
index 88d412c..b89aa56 100644
--- a/extras/glusterd.vol
+++ b/extras/glusterd.vol
@@ -9,6 +9,6 @@ volume management
     option event-threads 1
 #   option lock-timer 180
 #   option transport.address-family inet6
-#   option base-port 49152
+#   option base-port 59152
 #   option max-port  65535
 end-volume
-- 
2.17.1

